src/viur/core/email.py

```diff
@@ -209,76 +218,93 @@ def sendEMail(*,
         (for all text and attachments combined)!
     """
     # First, ensure we're able to send email at all
-    transportClass = conf.email.transport_class  # First, ensure we're able to send email at all
-    assert issubclass(transportClass, EmailTransport), "No or invalid email transportclass specified!"
+    transport_class = conf.email.transport_class  # First, ensure we're able to send email at all
+    assert issubclass(transport_class, EmailTransport), "No or invalid email transportclass specified!"
+
     # Ensure that all recipient parameters (dest, cc, bcc) are a list
     dests = normalize_to_list(dests)
     cc = normalize_to_list(cc)
     bcc = normalize_to_list(bcc)
+
     assert dests or cc or bcc, "No destination address given"
     assert all([isinstance(x, str) and x for x in dests]), "Found non-string or empty destination address"
     assert all([isinstance(x, str) and x for x in cc]), "Found non-string or empty cc address"
     assert all([isinstance(x, str) and x for x in bcc]), "Found non-string or empty bcc address"
-    attachments = normalize_to_list(attachments)
+
     if not (bool(stringTemplate) ^ bool(tpl)):
         raise ValueError("You have to set the params 'tpl' xor a 'stringTemplate'.")
-    if attachments:
+
+    if attachments := normalize_to_list(attachments):
         # Ensure each attachment has the filename key and rewrite each dict to db.Entity so we can exclude
         # it from being indexed
         for _ in range(0, len(attachments)):
             attachment = attachments.pop(0)
-            assert "filename" in attachment
+            assert all(att in attachment for att in ("filename", "content"))
```

Comment:
Please don't enforce the content. I've have a `TransportClass` implemention where u can provide just the file key.

```suggestion
            assert "filename" in attachment
```

Reply:
Ok, but then, this must generally be reworked, I think.

Reply:
Maybe we need a `transport_class.validate_attachment`... But I think this wasn't necessary before, so it shouldnt be a problem now either.

Reply:
Yes, please LGTM!